examples/compression/zlib.cpp

```diff
@@ -0,0 +1,34 @@
+#include <sincpp/compression/zlib.hpp>
+
+#include <iostream>
+
+int main() {
+  std::string_view const data = "Bouh!";
+  std::cout << "Intput = " << data << std::endl;
```

Comment:
Input ?

Reply:
I changed for "Data =" in both test and example. I also add the output as comment for the example.